packages/tokens/build.js

```diff
@@ -60,6 +120,36 @@ StyleDictionary.registerFormat({
   },
 })
 
+/** Creates named type declaration for Themes. Allows for TypeScript autocomplete */
+StyleDictionary.registerFormat({
+  name: 'typescript/es6-declarations/theme',
+  formatter: function (dictionary) {
+    let declaration = 'export declare type Theme = {\n'
+    dictionary.allProperties.forEach((token) => {
+      declaration += `  ${stripMode(token.name)}: string;\n`
+    })
+
+    declaration += `}
+
+export declare const themes: {
+  light: Theme;
+  dark: Theme;
+}`
+
+    return declaration
+  },
+})
+
+/** Custom format to generate index.d.ts with exports */
```

Comment:
Not sure if it's just an NPM issue, but getting an error (`This file failed to load`) trying to view the `index.d.ts` file in NPM.